tests/explicit-nulls/flexible-unpickle/Unsafe_1.scala

```diff
@@ -0,0 +1,21 @@
+package unsafeNulls
+
+class Unsafe_1 {
```

Comment:
We need to test more scala tests.
* `A => B`, `=> T`
* `A { val b: B }`, `A @B`
* `a.type`
* `(A, B)`
* `A | B`, `A & B`
* ...

Also, let's test not only the functions, but also constructors, members (value and type), extension methods, etc.